src/main/java/seedu/duke/Deadline.java

```diff
@@ -0,0 +1,61 @@
+package seedu.duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Holds the deadline task and relative operations.
+ * Inherits from Task class.
+ */
+public class Deadline extends Task {
+
+    /**
+     * Stores Time(Date) information of the deadline object
+     */
+    protected LocalDate time;
+
+    /**
+     * Initializes a deadline object.
+     * Sets the status to false(not done).
+     *
+     * @param content Content of the deadline object.
+     * @param time    Time of the deadline object, in string form, will be parsed to LocalDate.
+     */
+    public Deadline(String content, String time) {
+        super(content);
+        this.time = LocalDate.parse(time, DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+    }
+
+    /**
+     * Initializes a deadline object.
+     *
+     * @param status  Status of the deadline object.
+     * @param content Content of the deadline object.
+     * @param time    Time of the deadline object, int string form, will be parsed to LocalDate.
+     */
+    public Deadline(boolean status, String content, String time) {
```

Comment:
Perhaps having a more easily understandable variable name to make it sound more like a boolean?